tests/Unit/Parallel/CMakeLists.txt

```diff
@@ -131,7 +131,7 @@ function(add_algorithm_test_with_balancing TEST_NAME EXECUTABLE_NAME REGEX_TO_MA
       "\"Integration.Parallel.${TEST_NAME}\""
```

Comment:
Thanks for fixing this @moxcodes. Please also change this `Integration` -> `Unit` (another occurrence below)